extensions/wikia/AdEngine3/AdEngine3.setup.php

```diff
@@ -10,6 +10,7 @@
 // Autoload
 $wgAutoloadClasses['AdEngine3'] =  __DIR__ . '/AdEngine3.class.php';
 $wgAutoloadClasses['AdEngine3ApiController'] = __DIR__ . '/AdEngine3ApiController.class.php';
+$wgAutoloadClasses['AdEngine3EndpointController'] = __DIR__ . '/AdEngine3EndpointController.class.php';
```

Comment:
Not alphabetical ;_;

Reply:
They are related hence are grouped together.